dist/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/styles/style.css">
+    <title>Coat Or Shirt?</title>
+</head>
+
+<body>
+    <div class="container-fluid d-flex flex-column align-items-center pt-5">
+        <h1 class="mb-5">What should you wear today? 🤔</h1>
+        <form>
+            <div class="mb-3">
+                <input id="locationInput" type="text" class="form-control" placeholder="📍 Send me your Location 🗺️ (Song by Khalid)">
+                <div id="locationHelp" class="form-text">We'll never share your location with anyone else.</div>
+            </div>
+
+
+            <div class="form-check form-check-inline">
+                <input id="flexRadioDefault1"  class="form-check-input" type="radio" name="flexRadioDefault"checked>
+                <label class="form-check-label" for="flexRadioDefault1">
+                    Fahrenheit
+                </label>
+            </div>
+
+            <div class="form-check form-check-inline">
+                <input id="flexRadioDefault2" class="form-check-input" type="radio" name="flexRadioDefault">
```

Comment:
- [ ] [REQUIRED] Please use ` toggle control` instead of `radio button`.   ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))

Reply:
Added to future improvements. ✔️

Reply:
Done ✅